### ISSUE TITLE
Where is your Day 8 and What did you do with it?!

### ISSUE THREAD

**Issue by jrc-exp** [jenxuexp/advent2021#2]
**System information**

- OS Platform and Distribution: Linux Ubuntu 20.04
- TensorFlow version: 2.6
- Python version: 3.8.8
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): 9.3.0
- CUDA/cuDNN version: 11.5.0/8.3.0.96

**Question**

Look, none of us are likely proud of our day 8 solution. 

Serious and important question - did you solve part 2 essentially before submitting part 1?!? 

![image](https://user-images.githubusercontent.com/55510457/145218970-4bc17440-ece2-4a5f-9de5-61da86bd2f4f.png)

Otherwise your Delta Time is insane and would've been faster than the global winners.

**Comment by jenxuexp** [comment]
Yeah I solved part 2 first because of a reading comprehension mistake. I didn't realize all you had to do was get the unique numbers for part 1...

**Comment by jrc-exp** [comment]
https://github.com/mebeim/aoc/tree/master/2021#day-8---seven-segment-search

I believe you also haven't updated your README.md appropriately. See above for format guidelines.
